R/awsjavasdk-internal.R

```diff
@@ -17,8 +17,9 @@
 .app_dir <- rappdirs::app_dir("awsjavasdk", version = packageVersion("awsjavasdk"))
 
 .onAttach <- function(libname, pkgname) {
+  set_sdk_file_root()
   if (!aws_sdk_present()) {
-    packageStartupMessage("awsjavasdk did not detect its own copy of the AWS SDK; you must execute install_aws_sdk() in order to download the SDK and for the package to work as expected")
+    packageStartupMessage(paste0("awsjavasdk did not detect its own copy of the AWS SDK. You must execute install_aws_sdk() in order to download the SDK and for the package to work as expected. If you've previously installed to a non-standard location, call set_sdk_file_root({{your alt_install root}}, confirm = TRUE) to point this package to that location and then run awsjavasdk::load_sdk()"))
```

Comment:
@drknexus -- AWR is on CRAN after a few iterations: https://cran.r-project.org/package=AWR

Unfortunately, due to the repo design (see below), it was not possible to come up with a PR.

The "trick" (after some conversation with the CRAN maintainers) is that the package automatically downloads and installs the jars from the GitLab repository on first load (which is technically part of the installation), so end-users don't have to manually run a function to download the jars.

The related GitLab repository also stores all AWS SDK versions as per tags and the AWR package can be installed with any AWS SDK version thanks to this design. The only disadvantage of this approach is the pretty big repo size (currently around 6.5 gigs), but fortunately, GitLab allows that (although not possible on GitHub).

The first package using the jars bundled with `AWR` was also published on CRAN in the past few days at https://cran.r-project.org/package=AWR.KMS, and another package on Kinesis is to be pushed there as well after some refactoring.